dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css

```diff
@@ -8882,6 +8882,10 @@ div#_dotHelpMenu {
   content: "";
 }
 
+.globeIcon:before {
```

Comment:
Don't use "globe" use the semantic name: "languageVarIcon" or whatever the name is :)